com.github.micahflee.torbrowser-launcher.yaml

```diff
@@ -105,6 +105,21 @@ modules:
         url: https://files.pythonhosted.org/packages/bd/11/293dd436aea955d45fc4e8a35b6ae7270f5b8e00b53cf6c024c83b657a11/PySocks-1.7.1.tar.gz
         sha256: 3f8804571ebe159c380ac6de37643bb4685970655d3bba243530d6558b799aa0
 
+  - name: python3-packaging
+    buildsystem: simple
+    build-commands:
+      - "pip3 install --exists-action=i --no-index --find-links=\"file://${PWD}\" --prefix=${FLATPAK_DEST} \"packaging\""
```

Comment:
I think there don't have to be the backslashes in `\"packaging\"`, instead it can be just `packaging`. :-)